src/annotator/highlighter/overlay-highlighter/overlay-events.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const throttle = require('lodash.throttle');
+const overlayUtils = require('./overlay-utils');
+
+let _boundEventHandlers;
+
+const _invokeEvent = (eventData = {}, highlightElement, target)=>{
+  if(eventData.type in _boundEventHandlers){
+    _boundEventHandlers[eventData.type](Object.assign(eventData, {
+      currentTarget: highlightElement,
+      target: target || highlightElement,
+    }));
+  }
+};
+
+const _relayEventToHighlight = (e, highlightReferences) => {
```

Comment:
To recap, the purpose of this code is to simulate the sequence of events that would occur when clicking on a set of nested highlight elements (as created by the wrapping highlighter), when clicking on highlight elements created by the overlay highlighter?